src/components/onboarding/sign-in/index.js

```diff
@@ -0,0 +1,125 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React from 'react'
+import * as ReactRedux from 'react-redux'
+import ROUTE from 'constants/route'
+import PropTypes from 'prop-types'
+import InputField from 'components/common/input-field'
+import NextButton from 'components/common/next-button'
+import Icon from 'components/common/icon'
+import authActions from 'redux/actions/auth'
+import LinkButton from 'components/common/link-button'
+import style from './index.style.js'
+
+class SignIn extends React.Component {
+  state = {
+    email: '',
+    password: '',
+  }
+  handleChange = e => {
+    this.setState({
+      [e.target.id]: e.target.value,
+    })
+  }
+  handleSignIn = e => {
+    e.preventDefault() //  prevent reload of the page
+    this.props.signIn(this.state)
+  }
+
+  renderBackLink() {
+    return (
+      <div css={style.backLink}>
+        <LinkButton to={ROUTE.PATH.HOME}>
+          <div>
+            <Icon.LeftArrow />
+          </div>
+        </LinkButton>
```

Comment:
Wait for @andersonmmi to make a `<BackButton>` and use it here.
https://stackoverflow.com/questions/30915173/react-router-go-back-a-page-how-do-you-configure-history

Reply:
I've made a reusable back button in the [need-wallet] branch.